sos/report/plugins/networking.py

```diff
@@ -262,6 +250,17 @@ def setup(self):
             "netstat -s": "netstat_s"
         })
 
+    def collect_bridge_info(self):
+        """ Collect information about bridges (some data already collected via
+        "ip .." commands)
+        """
+        self.add_cmd_output([
+            "bridge -s -s -d link show",
+            "bridge -s -s -d -t fdb show",
+            "bridge -s -s -d -t mdb show",
+            "bridge -d vlan show"
+        ])
```

Comment:
Why move this to a dedicated function instead of leaving it in `setup()`? It's a single call with no gating logic.

Reply:
Was retrying to split/refactor a bit to fix too-many-{statements, locals, branches} warnings. You're right gating logic - aim was to split and thought "bridge" info could be one such independent part.